src/dialog/index.ts

```diff
@@ -160,8 +159,8 @@ export class DialogBase<P extends DialogProperties = DialogProperties> extends T
 		const { underlay } = this.properties;
 		return v('div', {
 			classes: [ this.theme(underlay ? css.underlayVisible : null), fixedCss.underlay ],
-			enterAnimation: animations.fadeIn,
-			exitAnimation: animations.fadeOut,
+			enterAnimation: this.theme(css.underlayEnter) || undefined,
```

Comment:
We should probably update `SupportedClassName` typings in `framework/widget-core` should be updated so this isn't necessary 😝